ADD doc for env vars (#3659)
telefonicaid/fiware-orion: doc/manuals/admin/cli.md

-   **-insecureNotif**. Allow HTTPS notifications to peers which certificate cannot be authenticated with known CA certificates. This is similar
    to the `-k` or `--insecure` parameteres of the curl command.

## Configuration using environment variables

Orion supports argument passing using environment variables. Each CLI parameter
has an equivalent environment variable, as shown in the table below
(`contextBroker -U` can be also used to get the same information).

> Two comments have to be taken into account:

Reviewer: ```suggestion
Two comments have to be taken into account:
Two facts have to be taken into account:
```

Author: Fixed commiting suggestion

* Environment variables for CLI parameters that work in a "flag" way 

>   (i.e. they are enabled or not, but doesn't have an actual value, such as `-fg`)

Reviewer: ```suggestion
  (i.e. they are either enabled or not, but don't have an actual value (`-fg` is one of them)
```

Author: Fixed commiting suggestion

  can take the case-sensitive values `TRUE` (to enable the parameter) or
  `FALSE` (to disable it).

> * In the case of conflict (i.e. using at the same time the environment variable

Reviewer: ```suggestion
* In case of conflict (i.e. using at the same time the environment variable
```

Author: Fixed commiting suggestion

>   and the CLI parameter) the later takes preference over the former.

Reviewer: ```suggestion
  and the CLI parameter) the CLI parameter is used.
```

Author: Fixed commiting suggestion
